test/flows/check_travel_during_coronavirus_flow_test.rb

```diff
@@ -365,6 +365,10 @@ class CheckTravelDuringCoronavirusFlowTest < ActiveSupport::TestCase
         assert_rendered_outcome text: "Returning to England if you're fully vaccinated"
       end
 
+      should "render guidance for people who are taking part in a vaccine trial" do
+        assert_rendered_outcome text: "Returning to England if you're fully vaccinated"
+      end
+
```

Comment:
If we're hiding links from people who are exempt, perhaps we should have a test for it?

Reply:
And have a test that the new content displays. 🥺

Reply:
I added a test for the new content displayed, its a bit further up the file `should "render 'exempt vaccination content' if exempt from vaccination" do`. 

I struggled a bit with testing hiding the link text because the text still displays on the page in the heading. The heading is updating in a future task to 'Returning to England if you’re fully vaccinated or cannot have a COVID-19 vaccination'  but I might be able to be more specific about there not being link text in the assertion... i'll have a look!

Reply:
Ah, sorry. I missed that the exemption test was in the previous commit 🤦‍♀️

Reply:
hehe that is ok! I was worried I forgot to push it

Reply:
@leenagupte I updated it with tests! Looking at it again I have no idea what I was on about about above about the text displaying elsewhere on the page... as that 'Returning to England...' text is completely different to the link text... so sorry about that 😱.